### PR TITLE
Config examples

### PR DESCRIPTION
**Pull request by lekterable**

### CODE REVIEW

File: packages/eslint-config-node/readme.md [hunk 1]
```diff
@@ -28,6 +28,46 @@ Use this ruleset in conjunction with any of the above version-specific rulesets.
 
 This ruleset includes rules which deal with how the code looks like and not how it works. It helps keeping the code clean and consistent. 🎨
 
+## STRV Config
+
+package.json
```
Comment (robertrossmann) [inline packages/eslint-config-node/readme.md]: I am personally a huge fan of explicit config files per tool. _package.json_ should be only used for declaring the package's requirements, not as a place to store configuration options.

File: packages/eslint-config-node/readme.md [hunk 2]
```diff
@@ -28,6 +28,46 @@ Use this ruleset in conjunction with any of the above version-specific rulesets.
...
+    "extends": [
+      "@strv/eslint-config-node/<v10 or v12>",
+      "@strv/eslint-config-node/optional",
+      "@strv/eslint-config-node/style"
+    ]
+  }
+}
+```
+
+.eslintrc or .eslintrc.json
```
Comment (robertrossmann) [inline packages/eslint-config-node/readme.md]: Let's switch to _.*rc.js_ examples wherever supported. 🙏 You can have comments in JS files and you can even add some tiny amount of scripting/dynamic behaviour if you really really need to.

File: packages/eslint-config-node/readme.md [hunk 3]
```diff
@@ -28,6 +28,46 @@ Use this ruleset in conjunction with any of the above version-specific rulesets.
...
+
+```json
+{
+  "extends": [
+    "@strv/eslint-config-node/<v10 or v12>",
```
Comment (robertrossmann) [inline packages/eslint-config-node/readme.md]: ESLint allows you to drop the `eslint-config-` prefix in ruleset names, so my recommendation would be to simply use `"@strv/node/v10"` and similar.

**Comment by dannytce** [comment]
Hi @robertrossmann! I just finally read your message mentioning tutorial docs. Damn, I've never seen it, thus I asked @lekterable to create these examples 😅

Agreed, that maybe examples should be placed here. But at the same time, we should have a link to them from each package.

**Comment by robertrossmann** [comment]
Yeah, perhaps the packages' readmes need some love in this regard. 👍

**Comment by dannytce** [comment]
@prichodko @robertrossmann @lekterable damn, looks like there was just stylelint missing in #38 🤔 

Anyways, I think now it's ready for the final review :)

**Comment by robertrossmann** [comment]
@dannytce I added some extra info into ESLint-related package documentation, have a look or just merge if you are okay with them! ✅

**Comment by dannytce** [comment]
@robertrossmann why did you removed `<details><summary>` examples of ESLint? :) I know that our preferred way is to use `.eslintrc.js`, but others may used it differently :) I would probably keep it there since it's just an example. And those are also collapsed by default.

**Comment by robertrossmann** [comment]
@dannytce they are back now.

**Comment by robertrossmann** [comment]
:beers: :tada:

Thanks @lekterable and @dannytce for contribution!

**Pull request merged**